src/ameba/ast/visitors/implicit_return_visitor.cr

```diff
@@ -0,0 +1,216 @@
+module Ameba::AST
+  class ImplicitReturnVisitor < BaseVisitor
+    # When greater than zero, indicates the current node's return value is used
+    @stack : Int32 = 0
+
+    def visit(node : Crystal::Expressions) : Nil
```

Comment:
`visit` methods ought to return `Bool`, not `Nil`.

ditto elsewhere

Reply:
I did this as I didn't want to use the native `accept_children` methods, and this was easier than adding a `false` to the end of every visit method